src/Resources/public/js/sonata_admin.js

```diff
@@ -0,0 +1,49 @@
+// Globalize jquery
+import jQuery from 'jquery';
+global.$ = jQuery;
+global.jQuery = jQuery;
+
+// Styles
+import "../scss/sonata_admin.scss"
+
+// Vendors
+import "admin-lte"
+import "bootstrap"
+
+import "jquery-form"
+import "jquery-ui"
+import "jquery.scrollto"
+import "jquery-slimscroll"
+
+import "x-editable/dist/bootstrap3-editable/js/bootstrap-editable.min"
+
+import "icheck"
+import "waypoints/lib/jquery.waypoints"
+import "waypoints/lib/shortcuts/sticky.min"
+import "select2"
+
+// Loading langugage files for select2
+let language = window.navigator.userLanguage || window.navigator.language;
+language = language.split("-")[0];
+import('select2/select2_locale_' + language + '.js')
```

Comment:
This will evaulated at runtime or at compile time?

Reply:
it should be on runtime

Reply:
let me check that

Reply:
This is executed on runtime.